HoloPipelines/cleanUp.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
Could be renamed to `cleanUpTestInputData.py`. Also this should be in the `test` directory as it really relates to testing. When you move it please remember to update any paths if needed.

Reply:
'cleanUp.py' removed as clean up is done within the fixture in test_pipelines.py